catalog/plugin/catalog-plugin-metacard-validation/src/test/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPluginTest.java

```diff
@@ -26,6 +26,8 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 import static org.mockito.Mockito.withSettings;
+import static ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin.INVALID_TAG;
```

Comment:
I usually tend _not_ to use the same constants as the class under test to ensure that the unit tests will fail if the values are (willingly or by mistake) changed in the code.